### PR TITLE
Strings for the DNS settings

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

File: src/ui/settings/ViewAdvancedDNSSettings.qml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+                    // Now bother user if the ip is invalid :)
+                    case VPNSettings.UserDNSInvalid:
+                        //% "Invalid IP address"
+                        ipInput.error = qsTrId("vpn.settings.userDNS.invalid")
+                        ipInput.valueInvalid = true;
+                        break;
+
+                    case VPNSettings.UserDNSNotIPv4:
+                        //% "We currently support only ipv4 IP addresses"
```
Comment (flodolo) [inline src/ui/settings/ViewAdvancedDNSSettings.qml:122]: As commented elsewhere, this should say `… IPv4 IP addresses`

File: src/ui/settings/ViewAdvancedDNSSettings.qml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+            // TODO: this should be a radiobox
+            VPNCheckBoxRow {
+                id: useLocalDNS
+                objectName: "settingMozillaDNSEnabled"
+                width: parent.width - Theme.windowMargin
+
+                //% "Use local DNS"
+                labelText: qsTrId("vpn.advancedDNSSettings.localDNS")
+                //% "Resolve websites using a DNS in your local network"
```
Comment (flodolo) [inline src/ui/settings/ViewAdvancedDNSSettings.qml:78]: I feel like "Resolve websites" is imprecise enough to trigger someone who knows what a DNS does, and to not help someone who doesn't.

Comment (lesleyjanenorton) [reply src/ui/settings/ViewAdvancedDNSSettings.qml:78]: @chrisrooneys what are your thoughts here?

Comment (chrisrooneys) [reply src/ui/settings/ViewAdvancedDNSSettings.qml:78]: @flodolo @lesleyjanenorton is "Access" technically accurate enough?

Comment (flodolo) [reply src/ui/settings/ViewAdvancedDNSSettings.qml:78]: It isn't. The accurate version would be `Resolve domain names using a DNS…`, or `Resolve website domain names using a DNS…` if we want to keep mentionining "websites".

Comment (chrisrooneys) [reply src/ui/settings/ViewAdvancedDNSSettings.qml:78]: Between those I think "Resolve website domain names using a DNS" covers both groups of users better.

Comment (lesleyjanenorton) [reply src/ui/settings/ViewAdvancedDNSSettings.qml:78]: thank you @flodolo & @chrisrooneys !

**Review by lesleyjanenorton — APPROVED** [review]

**Pull request merged**